web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -57,7 +57,7 @@ import {
 import { makeAdvancedSearchQueryForLabel } from 'shared/utils/advancedSearchLabelQuery';
 
 // eslint-disable-next-line no-restricted-imports -- FIXME
-import { ResourcesResponse } from 'teleport/services/agents';
+import { ResourceLabel, ResourcesResponse } from 'teleport/services/agents';
```

Comment:
Let's duplicate `ResourceLabel` within `UnifiedResources.tsx` instead of importing more stuff from `teleport` into `shared`.